### ISSUE TITLE
m[xc]doc: Gets confused by default values as vectors

### ISSUE THREAD

**Issue by ebknudsen** [McStasMcXtrace/McCode#821]
The output generated by mcdoc (and mxdoc) gets a bit garbled when the the parameter has a default value of a vector of values. Example from 
```mcdoc Pol_guide_mirror```

![image](https://user-images.githubusercontent.com/855956/60708026-60447080-9f0d-11e9-9099-9e73e9d7b6f9.png)

Granted this is not a common situation, and will likely change slightly with cogen3, but....

**Comment by climbcat** [comment]
Yeah, it looks like some of the comma-based regex statements fail in this case!

**Comment by climbcat** [comment]
The bug is most likely related to line 536 of <code>mccodelib.utils.py</code>, in the function <code>parse_params</code>:

<code>parts = [s.strip() for s in params_line.split(',')]</code>

**Comment by climbcat** [comment]
Should be fixed by 485d419171500e11e9eb5a4be59f4981c5d1645a and 4417767e6925df978e57c19891a670b3431242bb.

![image](https://user-images.githubusercontent.com/11196009/61374078-bca77880-a89b-11e9-8ea5-866fdaf96a0c.png)

**Comment by willend** [comment]
Fixed indeed
